src/components/big-button/big-button.jsx

```diff
@@ -1,9 +1,20 @@
+import React, {Component} from 'react';
+
 import './big-button.css';
 
-function BigButton(props) {
-    return(
-        <div className="BigButton">BigButton</div>
-    );
+class BigButton extends React.Component {
+    constructor(props){
+        super(props);
+    }
+
+    render() {
+        const classes = `BigButton ${this.props.enabled ? "enabled" : "disabled"}`;
```

Comment:
You could have a default state (enabled for example) and only add disabled class. Then, CSS code will only need one extra css rule for disabled. Use simple quotes (convention in the project)
```suggestion
        const classes = `BigButton ${this.props.enabled ? '' : 'disabled'}`;
```